internal/controller/controllers/crd_utils.go

```diff
@@ -185,3 +185,11 @@ func AddLabel(labels map[string]string, labelKey, labelValue string) map[string]
 	labels[labelKey] = labelValue
 	return labels
 }
+
+func GetLabel(labels map[string]string, labelKey string) (value string, exists bool) {
```

Comment:
I don't think this needs to be exported if the only user is still in the controllers package.

Reply:
OK